src/icons/annotations/IconHighlightText.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import AccessibleSVG from '../accessible-svg';
+import { white } from '../../styles/variables';
+import { Icon } from '../iconTypes';
+
+const IconHighlightText = ({
```

Comment:
Would it be more consistent with the existing `IconRegionAnnotation` if this was `IconHighlightTextAnnotation`?